runscope/README.md

```diff
@@ -0,0 +1,6 @@
+#Runscope API test results collector
+
+This script collects the API test results from Runscope.
+
+Usage:
+python runscope.py runscope-api-key
```

Comment:
Can you please add a requirements section and specify `pip install requests`. And also specify whether both Python 2.7+ and 3.x are supported.

Reply:
It seems the OS template which i used has this package preinstalled hence i missed this.  Tested this script with both 2.7 and 3.5 will add it worked will add the support comment as well.